ipn/ipnlocal/local.go

```diff
@@ -4605,6 +4605,10 @@ func (b *LocalBackend) FileTargets() ([]*apitype.FileTarget, error) {
 		if !b.peerIsTaildropTargetLocked(p) {
 			continue
 		}
+		if p.Hostinfo().OS() == "tvOS" {
+			fmt.Printf("don't show tvOS hosts")
```

Comment:
needed?

Reply:
oh no oops that was just for my local testing, i'll take it out!